### PR TITLE
Fixing offsets

### PR DESCRIPTION
**Pull request by LindsayYoung**
variable fix

### CODE REVIEW

**Comment by LindsayYoung** [comment]
This will require `manage.py update_schemas`

**Comment by LindsayYoung** [comment]
Checking the data on this

**Comment by ccostino** [comment]
Oh d'oh, looks like the test subset data needs to be fixed to account for this too. :-(

File: data/sql_updates/totals_pac.sql [hunk 1]
```diff
@@ -60,7 +60,7 @@ with last as (
         sum(pnp.net_op_exp_per) as net_operating_expenditures,
         sum(pnp.non_alloc_fed_elect_actvy_per) as non_allocated_fed_election_activity,
         sum(pnp.ttl_nonfed_tranf_per) as total_transfers,
-        sum(pnp.ttl_op_exp_per) as offsets_to_operating_expenditures,
+        sum(pnp.offests_to_op_exp) as offsets_to_operating_expenditures,
```
Comment (ccostino) [inline data/sql_updates/totals_pac.sql]: Ah I see one, `offsets` :-)

File: data/sql_updates/reports_pac_party.sql [hunk 2]
```diff
@@ -45,8 +45,8 @@ select
     non_alloc_fed_elect_actvy_per as non_allocated_fed_election_activity_period,
     non_alloc_fed_elect_actvy_ytd as non_allocated_fed_election_activity_ytd,
     shared_nonfed_op_exp_per as nonfed_share_allocated_disbursements_period,
-    ttl_op_exp_per as offsets_to_operating_expenditures_period,
-    ttl_op_exp_ytd as offsets_to_operating_expenditures_ytd,
+    offests_to_op_exp as offsets_to_operating_expenditures_period,
```
Comment (ccostino) [inline data/sql_updates/reports_pac_party.sql]: `offsets`

File: data/sql_updates/reports_pac_party.sql [hunk 3]
```diff
@@ -45,8 +45,8 @@ select
...
+    offsets_to_op_exp_ytd_i as offsets_to_operating_expenditures_ytd,
```
Comment (ccostino) [inline data/sql_updates/reports_pac_party.sql]: Is the `_i` at the end intended?

Comment (LindsayYoung) [inline data/sql_updates/reports_pac_party.sql]: yea, and it means that we have a problem in the totals, I documented in the issue.

The problem is that on the forms they ask the same question more than once, but people don't always fill out things twice, so you need to take the greatest of both values.

That was the way on the original table, then we switched to a case where that wasn't true, Then we switched to a case where that was true again, but did not add the greatest back in.

**Comment by ccostino** [comment]
Who would've thought the spelling things correctly would be an issue... 😆 ❤️ thanks @LindsayYoung!

**Pull request merged**